### DISCUSSION TITLE
8470p post installation reboot doesn't finish and repeat

### DISCUSSION THREAD

**Discussion by betek2k** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#158] [Q&A]
I followed this guide step by step to install macOS Monterey 12.6.01 for my HP Elitebook 8470p (i5-3320M 2.6Ghz, Hi-rez 1600x900);

*Update BIOS to ver.F74
*Setup BIOS accordingly to this guide
*I watched all of the videos here and successfully created a USB installer and finished installing Monterey on my 8470p.
Of course, at this point, nothing really works other than the trackpad and keyboard. Then I followed the post-install guide video, using the post-installer and "Patch HD4000" and after everything is done, I rebooted it, crossing my fingers. In the video, after the reboot, everything worked, and happy ending, but unfortunately in my case, the reboot started and in the middle of it, it stopped and repeat from the beginning again and again. I tried many times by erasing the drive, reinstalling the Monterey again, and applying post installer, Patch HD4000, but always come to the same result. Will someone please give me any advice that I may be doing something wrong? One thing I'm wondering about is the "Disabling the security SIP" thing. When I boot with the USB installer and came to the recovery window, I used the terminal and typed "csrutil disable" but I got the error message "failed to update system integrity configuration..." so then I typed "csrutil status", then I see SIP is disabled. (Custom Configuration). Chris is not doing anything related to SIP on the video, so I'm not so sure. I used the "gatekeeper disabling script" as he did on the video, though. Another thing is that my Wifi card is not listed in the "Recommended Wifi". Mine is "Broadcom BCM943225HMB". Can something like this cause a boot loop? I hope someone can help me out. Thank you for your time.
![IMG_1273](https://user-images.githubusercontent.com/117395332/200254253-77142321-f37c-4523-9a7d-6e885281bfab.jpeg)
![IMG_1282](https://user-images.githubusercontent.com/117395332/200254305-1886646b-0be8-42d4-9b4f-fd8b4c0656d5.jpeg)

**Comment by chris1111** [comment]
You ca try to apply the same boot args recommanded for the BCM943224HMS
`brcmfx-delay=15000`
Try then let me known

**Reply by betek2k** [reply]
Thanks for the reply, Chris. I really appreciate your work.
I added the boot-args "brcmfx-delay=15000" to my config.plist in the EFI partition of my SSD, but unfortunately, it did not work. Same results. BTW, I formatted my SSD as APFS/GUID for installing Monterey. It's Okay, right?
![IMG_1284](https://user-images.githubusercontent.com/117395332/200415177-7615dd6a-ed80-4b8a-b401-2988a8396126.jpeg)
[config.plist.zip](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/files/9955460/config.plist.zip)

**Comment by chris1111** [comment]
Yes Right

**Comment by chris1111** [comment]
Yes Right but -igfxvesa is not to be used after post installation

**Reply by betek2k** [reply]
OK, Thanks for clarifying it. I'll remove "-igfxvesa" boot-args from config.plist.

**Comment by betek2k** [comment]
Umm...I removed it but the same result. Thanks for your advice and time. 
Is there anything, or any information you might need to investigate the issue? config.plist, install log file?

**Comment by chris1111** [comment]
Isuue is from the Wireless card ;probably incompatible?

**Reply by betek2k** [reply]
I've been thinking about what could cause this and my suspects are 1. SIP, 2. Wifi card 3. Disk format (APFS/GUID), 4. Monterey ver. 12.6.1. So I mentioned them here and after I read your reply, I came to the same conclusion as yours which is the old incompatible WiFI card. (I had this card successfully working on my 8470p Hackintosh with Mojave/Clover).
So at this point what can I do to investigate? If I remove the Wifi card from 8470p, Do you think at least I can finish rebooting? Of course, WiFi would not work but at least other things might work like monitor resolution, etc? Or just wait until I actually get the recommended WiFi card and installed it on my 8470p?

**Comment by chris1111** [comment]
If you remove the card and use correctly the program, this will be work

**Reply by betek2k** [reply]
OK, Cool! I'll try that and let's see what happens. Thank you so much for your advice. I was so lost and almost giving this up but now I have hope! I just ordered AzureWave BCM94352HMB WiFi card on eBay, too. I hope macOS Ventura will work on Ivy-Era Probook/EliteBook with your OC package near future!

**Reply by betek2k** [reply]
After I removed the incompatible old WiFi card (Broadcom BCM943225HMB) from 8470p, and run the post-installer again with unchecked "AirportBrcmFixup" and "Bluetooth Fix", also run the "PatchHD4000" again and reboot, and Voilà! It worked!
Of course, except for WiFi and Bluetooth. But now, I know for sure the problem was caused by the incompatible old WiFi card.
My 1600x900 monitor looks perfect now! As soon as I get a new WiFi card, everything will be working fine. 
Thank you so much, Chris!
